packages/ag-charts-community/src/chart/interaction/cursorManager.ts

```diff
@@ -1,6 +1,20 @@
 import { StateTracker } from '../../util/stateTracker';
 import type { DOMManager } from '../dom/domManager';
 
+export enum Cursor {
```

Comment:
Contentious point: I'm actually against `enum` for the sake of `enum` - I think this would be better as a simple string-union type; TS `enum`s don't really have any strong benefit, in fact they add an extra layer of abstraction which reduces readability IMO.

You can achieve the same efficiency of declaration using `as const`:
```
export const CURSORS = ['auto', 'default'] as const;
export type Cursor = typeof CURSOR_TYPES[number];
```

Reply:
I'm of the opposite opinion. I think the greatest benefit comes when refactoring + you always have the context of the value attached to it whereas seeing `'top'` or `'auto'` can be very confusing and out of place in case a few entities are involved.

Reply:
It was simply because sonarqube was complaining about repeated strings, and adding a `const CursorNotAllowed = 'not-allowed';` in my particular case seemed a bit meh since it is a shared component. But can just do that if you prefer?

Reply:
And yeh, I agree with Ido, `as const` is very hacky to my eyes